pkg/apis/servicebinding/v1alpha3/servicebinding_test.go

```diff
@@ -442,7 +456,7 @@ func TestServiceBindingStatus_MarkServiceAvailable(t *testing.T) {
 			{
 				Type:               ServiceBindingConditionReady,
 				Status:             metav1.ConditionUnknown,
-				Reason:             "Unknown",
+				Reason:             "ProjectionReadyUnknown",
```

Comment:
Is that Reason `ProjectionReadyUnknown` correct? Can you please help me with some detail for this reason value?

Reply:
aggregare func builds the reason string with sub condition type + condition reason. Same for other comment